### PR TITLE
LPS-176598 Layou filter created and applyed on table and updated translations

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:relevant

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts [hunk 1]
```diff
@@ -547,9 +570,18 @@ const filterSchema = {
...
 				options: [
-					{label: 'Complete', value: SubTaskStatuses.COMPLETE},
-					{label: 'In Analysis', value: SubTaskStatuses.IN_ANALYSIS},
-					{label: 'Open', value: SubTaskStatuses.OPEN},
+					{
+						label: i18n.translate('complete'),
+						value: SubTaskStatuses.COMPLETE,
+					},
+					{
+						label: i18n.translate('in-abalysis'),
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts:578]:
```suggestion
						label: i18n.translate('in-analysis'),
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Pull request closed without merging**